### PR TITLE
CU-grrmr9_Delete-confirmation

### PR DESCRIPTION
**Pull request by bmitch242**
Delete confirmation alert works locally, don't have my server stuff set up yet but I tested it in a different place for syntax

### CODE REVIEW

**Comment by bmitch242** [comment]
Task linked: [CU-grrmr9 Delete confirmation](https://app.clickup.com/t/grrmr9)

**Review by carteralbrecht — CHANGES_REQUESTED** [review]
I think the room card rendering is commented out

File: mobile/src/screens/Dashboard.js [hunk 1]
```diff
@@ -129,7 +139,7 @@ class Dashboard extends React.Component {
           />
         </View>
             <Header2 title="Chat List:"/>
-
+{/*}
```
Comment (carteralbrecht) [inline mobile/src/screens/Dashboard.js:142]: Intentional?

Comment (bmitch242) [reply mobile/src/screens/Dashboard.js]: My bad, should be fixed now

**Review by carteralbrecht — APPROVED** [review]

**Pull request merged**